tron/serialize/runstate/dynamodb_state_store.py

```diff
@@ -153,14 +167,15 @@ def _merge_items(self, first_items, remaining_items, read_json=False) -> dict:
         if remaining_items:
             first_items.extend(remaining_items)
         for item in first_items:
-            key = item["key"]["S"]  # key column of an item = "job_State MASTER.cits_test_load_foo1"
-            items[key].append(item)  # for jobs state one item is basically info for each job: run nums and enabled
+            key = item["key"]["S"]  # example of a key for a job name "job_State MASTER.cits_test_load_foo1"
```

Comment:
nit: `# example of a key for a job name: "job_state MASTER.cits_test_load_foo1"`

that said - the key name format doesn't actually matter right? maybe we could move this comment elsewhere (or remove it?)

Reply:
yea, i had it earlier as a reference to myself when i was looking at the code for what a key would look like